### PR TITLE
remove all references to tiebreaking groovy scripts

### PR DESCRIPTION
**Pull request by missinglink**
FTW! remove all references to tiebreaking groovy scripts

closes: https://github.com/pelias/scripts/issues/12

### CODE REVIEW

File: test/unit/fixture/search_linguistic_bbox.js [hunk 1]
```diff
@@ -86,7 +86,7 @@ module.exports = {
       }
     }
   },
-  'sort': [ '_sort' ],
```
Comment (missinglink) [inline test/unit/fixture/search_linguistic_bbox.js]: these were typos in the fixtures, masked by some code which I just deleted

**Comment by missinglink** [comment]
when merging this PR please make sure the documentation is updated and ticket created for updating the vagrant scripts and mapzen search deployment scripts.

**Comment by missinglink** [comment]
discussed in more detail [here](https://github.com/pelias/scripts/issues/12), this will have 0 impact on results

**Comment by riordan** [comment]
👏

**Comment by missinglink** [comment]
update readmes:
- https://github.com/pelias/openstreetmap/commit/a4f9a3bd44b1c68c29cfbf8892a6ce5bd3ff8a80
- https://github.com/pelias/api/commit/687c26555fdab7cc65738c82f483a6149b57065e

**Pull request merged**